src/NServiceBus.AcceptanceTests/Sagas/When_a_finder_exists_and_found_saga.cs

```diff
@@ -11,7 +11,7 @@
 
     [TestFixture]
     public class When_a_finder_exists_and_found_saga : NServiceBusAcceptanceTest
-    {
+    {        
```

Comment:
:fire: 

I'd recommend installing the [Trailing Whitespace Visualizer](https://visualstudiogallery.msdn.microsoft.com/a204e29b-1778-4dae-affd-209bea658a59) VS plugin.